draft-yasskin-http-origin-signed-responses.md

```diff
@@ -832,17 +822,16 @@ signature returns "valid", return "valid". Otherwise, return "invalid".
    [same-origin](https://html.spec.whatwg.org/multipage/origin.html#same-origin)
    with `requestUrl`, return "invalid".
 1. Use {{signature-validity}} to determine the signature's validity for
-   `requestUrl`, `headers`, and `payload`, getting `certificate-chain` back. If
+   `requestUrl`, `responseHeaders`, and `payload`, getting `certificate-chain` back. If
    this returned "invalid" or didn't return a certificate chain, return
    "invalid".
-1. Let `exchange` be the exchange metadata and headers parsed out of `headers`.
-1. If `exchange`'s request method is not safe (Section 4.2.1 of {{!RFC7231}}),
```

Comment:
I think we ban POST method by this "not safe" check. (POST is actually cachable per https://tools.ietf.org/html/rfc7231#section-4.2.3 ). Should we keep this check?

Reply:
This line did ban POST, but since the headers can't encode a method anymore, I don't think we need to keep it.

Reply:
Ah I missed that. Thanks for the explanation :)